### DISCUSSION TITLE
Unable to use "&&" in SSR

### DISCUSSION THREAD

**Discussion by royalswe** [vikejs/vike#876] [Discussion]
I use the "&&" operator in a block of code and the i get the Hydration mismatch error.
Example:
```
const content = defineComponent({
  setup() {
    return () => [
      props.item.icon &&
      h('div', { class: 'material-symbols-outlined', innerHTML: props.item.icon })
    ]
  }
});
```

if I change && to a Conditional (ternary) operator it will work.
like this:
```
const content = defineComponent({
  setup() {
    return () => [
      props.item.icon ?
      h('div', { class: 'material-symbols-outlined', innerHTML: props.item.icon }) : undifined
    ]
  }
});
```

Is there a setting that has to be changed? like ecma version?

**Comment by brillout** [comment]
From reading your code snippets, I don't see how this can correlate with a hydration mismatch.

**Comment by royalswe** [comment]
This code was from a npm package I can manage, when i copied the code to my project it worked. Weird.
I don´t think we will find the issue easily so I just update that part with a ternary and go on.
